docs/content/docs/connectors/pipeline-connectors/elasticsearch.md

```diff
@@ -164,6 +164,13 @@ Pipeline Connector Options
       <td>Long</td>
       <td>The maximum size of a single record in bytes.</td>
     </tr>
+    <tr>
+      <td>sharding.suffix.key</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Long</td>
+      <td>Sharding suffix key for each table, allow setting sharding suffix key for multiTables.Default sink table name is test_table${suffix_key}.Tables are separated by ';'.For example, we can set sharding.suffix.key by 'table1:col1;table2:col2'.</td>
```

Comment:
Hi @beryllw, could you explain more about target scenario of this issue? I don't understand the meaning of this format of table name.